src/hapi.erl

```diff
@@ -480,3 +484,22 @@ transport_opts(tcp, Family) ->
 -spec path_query(uri()) -> string().
 path_query(URI) ->
     uri_string:recompose(maps:without([scheme, host, port, userinfo], URI)).
+
+-spec trace_request(req(), req_opts()) -> _.
```

Comment:
Actually I'd prefer for post-DNS resolution log messages to stay. They're useful for troubleshooting in integration phase: you can clearly see actual IP address used for HTTP-request

Reply:
My bad, original message was about __pre__-DNS log message `Looking up address` which is not useful with current tracing feature because trace messages already have hostname of current request. Removed such log.